snap/local/systemd/override.conf

```diff
@@ -0,0 +1,11 @@
+[Unit]
```

Comment:
Snapd generates a service file for the daemon (opensearch in this case), so ideally we would just need to edit it - but it seems like it's not [advised](https://github.com/snapcore/snapd/blob/83a4be185fa941a09ba27d252db4cf5666e62fbb/wrappers/services.go#L1133) to do so, and probably not safe to do so as this file gets overridden on upgrades, so the trick would be to have an override conf and reload the unit. We could potentially register a refresh hook to this, but I'm not entirely sure in which other case this might get overriden

Reply:
It turns out the properties I had set there can be replaced with ulimit commands, so I removed it